jdbc/src/main/java/org/apache/metamodel/jdbc/dialects/AbstractQueryRewriter.java

```diff
@@ -460,4 +460,25 @@ public Object getResultSetValue(ResultSet resultSet, int columnIndex, Column col
         }
         return resultSet.getObject(columnIndex);
     }
+
+    protected boolean isSupportedDatabase(String databaseProductName, int databaseVersion) {
```

Comment:
Shouldn't this be limited to `isSupportedVersion`? I mean, hopefully the concrete rewriters know what product they are.